core/src/main/java/de/bwaldvogel/mongo/backend/AbstractMongoCollection.java

```diff
@@ -469,7 +469,7 @@ public QueryResult handleQuery(QueryParameters queryParameters) {
         return writeErrors;
     }
 
-    private static Document toErrorDocument(MongoServerError e, int index) {
+    public static Document toErrorDocument(MongoServerError e, int index) {
```

Comment:
`protected` would be enough.
Please add the `@VisibleForExternalBackends` to indicate why it’s not a private method.

Reply:
`@VisibleForExternalBackends` added.